stubs/openssl-python/OpenSSL/SSL.pyi

```diff
@@ -0,0 +1,26 @@
+from socket import socket
+from typing import Callable, Optional, Sequence, Union
+
+from OpenSSL.crypto import X509
+
+TLSv1_2_METHOD: int = ...
+OP_NO_SSLv2: int = ...
+OP_NO_SSLv3: int = ...
+OP_NO_TLSv1: int = ...
+OP_NO_TLSv1_1: int = ...
+OP_NO_TLSv1_2: int = ...
+OP_NO_TLSv1_3: int = ...
+VERIFY_PEER: int = ...
+
+class Connection:
+    def __init__(self, context: Context, _socket: Optional[socket] = ...) -> None: ...
+    def connect(self, addr: Union[str, bytes, Sequence[Union[str, int]]]) -> None: ...
```

Comment:
We now prefer PEP 604 unions outside of type aliases, e.g. `socket | None` or `str | int`.